src/rgw/rgw_data_sync.cc

```diff
@@ -1706,6 +1708,20 @@ int RGWArchiveSyncModule::create_instance(CephContext *cct, const JSONFormattabl
   return 0;
 }
 
+void RGWArchiveDataSyncModule::init(RGWDataSyncEnv *sync_env, uint64_t instance_id) {
+
+  ldout(sync_env->cct, 0) << "Switching to archive zone metadata manager" << dendl;
+
+  // FIXME: this code crashes... migrating the current state (handles, md_logs,
```

Comment:
This code is legacy. We should also eliminate the support methods for metadata manager and clean it.